CONTRIBUTING.md

```diff
@@ -10,10 +10,10 @@ This project adheres to the [Open JS Foundation Code of Conduct](https://eslint.
 
 Before filing an issue, please be sure to read the guidelines for what you're reporting:
 
-* [Report Bugs](https://eslint.org/docs/latest/contribute/report-bugs)
-* [Propose a New Rule](https://eslint.org/docs/latest/contribute/propose-new-rule)
-* [Propose a Rule Change](https://eslint.org/docs/latest/contribute/propose-rule-change)
-* [Request a Change](https://eslint.org/docs/latest/contribute/request-change)
+*   [Report Bugs](https://eslint.org/docs/latest/contribute/report-bugs)
+*   [Propose a New Rule](https://eslint.org/docs/latest/contribute/propose-new-rule)
+*   [Propose a Rule Change](https://eslint.org/docs/latest/contribute/propose-rule-change)
+*   [Request a Change](https://eslint.org/docs/latest/contribute/request-change)
```

Comment:
These two extra spaces is a bug in Prettier? https://github.com/prettier/prettier/issues/5019

Reply:
oh, can I fix it by forcing tab indentation to 2?

Reply:
We'd prefer 4, perhaps we could wait to see if the bug might be fixed soon: https://github.com/prettier/prettier/pull/15526

Reply:
for now i'll change to 2, it looks like that PR has been sitting for a while